taskcluster/async/asyncclient.py

```diff
@@ -38,7 +38,10 @@
 
 def createSession(*args, **kwargs):
     """ Create a new aiohttp session.  This passes through all positional and
-    keyword arguments to the aiohttp.CreateSession() constructor
+    keyword arguments to the asyncutils.createSession() constructor.
+
+    `async with aiohttp.ClientSession(...) as session:` in the client code
```

Comment:
Can you elaborate on this snippet to create a client instance and call a `ping()` method?

Reply:
Oh, and I was thinking it might be nice to re-export `aiohttp.CreateSession()` from this library so that users only have to import taskcluster-client.py directly.

Reply:
Sure. Added a larger couple of examples. Amended `aiohttp.CreateSession` to just using `createSession`, which works as well.